.github/workflows/run-coverage.yml

```diff
@@ -56,5 +56,10 @@ jobs:
 
     - name: Run and report code coverage
       run: |
-        coverage run -m pytest
-        coverage report -m
\ No newline at end of file
+        pytest --cov=. --cov-report json
```

Comment:
I think `--cov=blah` overrides the `source` option in the pyproject.toml file, so if you want to specify the desired folder(s) in that file we should just put `--cov` here (with no equals sign or path).

On that note, I see from Codecov that it's including the `docs` folder, which makes sense given that we put `.` here. We definitely don't want to include `docs`, so maybe it would be better to just set `source` to `src` and not have to exclude `docs`, `tests`, etc.